src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
Everything accessible - not just everything documented - is public API, even if it has a magical underscore in the front of the property name.